package.json

```diff
@@ -34,30 +31,16 @@
 	"engines": {
 		"node": ">=v20"
 	},
-	"prettier": {
-		"tabWidth": 2,
-		"semi": false,
-		"singleQuote": true
-	},
-	"lint-staged": {
```

Comment:
still need lint-staged config, but using biome instead of eslint/prettier

Reply:
ok, bringing it back.